solidity/contracts/Coordinator.sol

```diff
@@ -1,11 +1,15 @@
 pragma solidity ^0.4.24;
 
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "./interfaces/LinkTokenInterface.sol";
 
 // Coordinator handles oracle service aggreements between one or more oracles.
-contract Coordinator {
+contract Coordinator is Ownable {
```

Comment:
I really don't think that we should be implementing Ownable for the Coordinator contract at any point.

Reply:
Thanks for all the feedback, Thomas. I removed this.